### PR TITLE
Flake 8

### PR DESCRIPTION
**Pull request by RussTorres**
enforces linewidth (except where it would screw up docstring formatting) and conforms the library and tests to a loose definition of flake8 standard such that my fresh new flake8 linter is not reporting any errors or warnings.

### CODE REVIEW

**Comment by RussTorres** [comment]
I think the lines removed are unused imports as I did some cleanup on that front as well.  resolves #92

File: integration_tests/test_pointmatch_integrated.py [hunk 1]
```diff
@@ -181,22 +176,23 @@ def test_get_match_groupIds_to_only(render, test_pm_collection):
...
     renderapi.pointmatch.delete_point_matches_between_groups(
         collection, '0', '1', render=render)
     groups = renderapi.pointmatch.get_match_groupIds(collection, render=render)
     assert len(groups) == 2
 
+
 def test_delete_collection(render):
     collection = 'test_delete_collection'
-    owner = 'test'
+    # owner = 'test'
```
Comment (fcollman) [inline integration_tests/test_pointmatch_integrated.py]: why aren't we just nuking these lines?

File: integration_tests/test_stack_integrated.py [hunk 2]
```diff
@@ -306,10 +305,10 @@ def test_bb_image(render, teststack, **kwargs):
         data = render.run(renderapi.image.get_bb_image,
                           teststack, z, x, y, width, height,
                           scale=.25, img_format=fmt, **kwargs)
-        dr = data.ravel()
+        dr = data.ravel()  # noqa: 841
```
Comment (fcollman) [inline integration_tests/test_stack_integrated.py]: this should be nuked.

File: renderapi/client.py [hunk 3]
```diff
@@ -935,7 +938,7 @@ def get_canvas_url_template(
     excludeAllTransforms: bool
         alternative to normalizeForMatching which simply removes all transforms from the list.
         default=False
-    """
+    """  # noqa: E501
```
Comment (fcollman) [inline renderapi/client.py]: what is the QA way of doing this?

**Comment by fcollman** [comment]
the only other comment I have is that I want you to show me how I can setup my editor so I get the same complaints that you do and I'll be better about keeping the flake8 up to snuff.

File: integration_tests/test_pointmatch_integrated.py (hunk 1 above)
Comment (RussTorres) [reply integration_tests/test_pointmatch_integrated.py]: Yeah, I wanted to ask about those... if you don't think they're useful they can go.

**Comment by RussTorres** [comment]
@fcollman I am happy with this.  Shall we merge?

**Comment by fcollman** [comment]
Yes!

**Pull request merged**